bricks/basic-bricks/src/editor-bricks/easy-view/easy-view.editor.tsx

```diff
@@ -27,6 +28,13 @@ export function EasyViewEditor({
   nodeUid,
 }: EditorComponentProps): React.ReactElement {
   const node = useBuilderNode<EasyViewProperties>({ nodeUid });
+  const { nodes, edges } = useBuilderData();
+  const parentUid = edges.find((item) => item.child === node.$$uid)?.parent;
+  const parentNode = nodes.find((item) => item.$$uid === parentUid);
```

Comment:
`@next-core/editor-bricks-helper` 有相关 hook：

```suggestion
  const parentNode = useBuilderParentNode(node.$$uid);
```